com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/Editor/ShaderGraph/BuiltInProperties.cs

```diff
@@ -71,7 +71,7 @@ internal static class Keyword
 
     internal static class BuiltInMaterialInspectorUtilities
     {
-        internal static void AddFloatProperty(this PropertyCollector collector, string referenceName, float defaultValue, HLSLDeclaration declarationType = HLSLDeclaration.DoNotDeclare)
+        internal static void AddFloatProperty(this PropertyCollector collector, string referenceName, float defaultValue, HLSLDeclaration declarationType = HLSLDeclaration.DoNotDeclare, bool generatePropertyBlock = true)
```

Comment:
is this the default behavior? Wouldn't it have been default false before because of `default` boolean being false?

Reply:
No, because the default value of the `generatePropertyBlock` boolean is true in the ShaderInput class: https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.shadergraph/Editor/Data/Graphs/ShaderInput.cs#L212
Setting this to false would have broken all the other properties.